src/julia_gc.c

```diff
@@ -196,7 +205,7 @@ static inline void * align_ptr(void * p)
     return (void *)u;
 }
 
-#if !defined(DISABLE_BIGVAL_TRACKING)
+#if !defined(SCAN_STACK_FOR_MPTRS_ONLY)
```

Comment:
... seems it's a good thing I insisted on putting my `DISABLE_BIGVAL_TRACKING` mod in there... ;-)